apps/docs/components/BodyContentTypeDropdown.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  IconChevronDown,
+} from 'ui'
+import { useState } from 'react'
+
+type IParamProps = any
+
+const BodyContentTypeDropdown = ({ types, onSelect }: IParamProps) => {
+  const [selectedType, setSelectedType] = useState(types[0] || '')
+  const myOnSelectType = (type: string) => {
+    setSelectedType(type)
+    if (onSelect) onSelect(type)
+  }
+
+  if (!types || types.length === 0) {
+    return <></>
+  }
+
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger>
+        <div
+          className="
+          group
+          justify-between
+          bg-control
+          border
+          hover:border-control
+          hover:bg-overlay-hover
+          border-control px-2 h-[32px] rounded
+          font-mono
+          flex items-center gap-1 text-foreground-muted text-xs group-hover:text-foreground transition
+          "
+        >
+          <span className="text-foreground text-sm group-hover:text-foreground transition">
```

Comment:
not massively important, but don't need a `<span>`

move these classes to `<div>` above.